packages/template.rb.template

```diff
@@ -80,4 +76,9 @@ class Template < Package
   end
 
 
-end
+  # Function to perform check from source build.
+  # This is execute if and only if `crew build`.
+  def self.check
```

Comment:
I arranged these in chronological sequence.  I like to have the sequence in packages too but so many packagers like to change it around. :)

Reply:
Reverted to the original sequence